product/views.py

```diff
@@ -9,6 +9,22 @@
 
 from django.shortcuts import render
 
+
+class MainView(View):
+    def get(self, request):
+        products = Product.objects.all()[0:6]    # 프로덕트 중에서 6개만 뽑음
```

Comment:
리미트 값을 준다면 하드코딩이 아니라 
PRODUCT_MAIN_LIMIT = 6
과 같이 상수로 지정하고 처리하는게 가독성에 좋습니다.
```suggestion
        products = Product.objects.all()[:PRODUCT_MAIN_LIMIT]    # 프로덕트 중에서 6개만 뽑음
```

Reply:
수정 사항 반영하였습니다.